tests/func/test_get.py

```diff
@@ -184,3 +185,30 @@ def test_get_from_non_dvc_master(tmp_dir, erepo_dir, caplog):
 
     assert caplog.text == ""
     assert (tmp_dir / dst).read_text() == "some_contents"
+
+
+def test_get_url_positive(tmp_dir, erepo_dir, caplog):
+    with erepo_dir.chdir():
+        erepo_dir.dvc_gen("foo", "foo")
+
+    caplog.clear()
+    with caplog.at_level(logging.ERROR, logger="dvc"):
```

Comment:
I think that in this test, you should set caplog level to `INFO`, and verify that proper message is displayed 
(In your case it will be `relpath` to `erepo` + ".dvc/cache" + "checksum_part"))
where checksum part is checksum[:2]/checksum[2:] (thats how we store files in cache).

This is important because it is testing the core of what this change is all about - informing the user what is the resource path. I would also consider displaying an absolute path, but we can leave this discussion for some other time.

Reply:
Honestly, I did not want to delve into how `dvc.api.get_url()` works previously, and verify it's correctness. I just wanted it to be delegated to `get_url()` and be its concern. I wanted this to be checked in unit tests, but, I quite missed to check the `caplog` there.

Would you still like to have a functional test for this? Or, are you okay with changes to the unit test? What do you think?

Reply:
@skshetry you are right about delegating work to `get_url`. I would be ok with patching `get_api` in unit test and verifying that proper value has been logged. As I see its already done, so lets just add caplog there and verify that proper value is displayed.

Reply:
Fixed by 02a73a0